sphinxcontrib/confluencebuilder/config/checks.py

```diff
@@ -316,6 +316,12 @@ def validate_configuration(builder):
 
     # ##################################################################
 
+    # confluence_table_width
+    validator.conf('confluence_table_width') \
+             .int_(positive=True)
```

Comment:
Instead of just an integer value, maybe an int or a string value should be accepted for various standard Sphinx length types used, and we use the [`extract_length`](https://github.com/sphinx-contrib/confluencebuilder/blob/v2.14/sphinxcontrib/confluencebuilder/util.py#L195)/[`convert_length`](https://github.com/sphinx-contrib/confluencebuilder/blob/v2.14/sphinxcontrib/confluencebuilder/util.py#L118) utility calls to resolve the final pixel value.

Reply:
I did my best to make it a int/string. Hope my solution is good enough.